openapi/schema/request/observation_photos_create.js

```diff
@@ -0,0 +1,12 @@
+const Joi = require( "@hapi/joi" );
+
+module.exports = Joi.object( ).keys( {
+  fields: Joi.any( ),
+  observation_photo: Joi.any( ),
+  // TODO Figure out how to get nested param validation to work. Currently it
```

Comment:
Is this why most of the create method request parameters use simply `Joi.object( )` instead of specifying the properties we will accept for an object? If so I think we'll need to address this before we have people using this so we can clearly specify the properties available to populate

Reply:
Yeah. That was pretty early on in the process, though, so maybe it would work now. I agree it's important to figure out a solution there before actual release.